website/content/docs/agent/wal-logstore/index.mdx

```diff
@@ -45,4 +47,12 @@ The WAL backend has been tested thoroughly during development:
 
 We are aware of how complex and critical disk-persistence is for your data.
 
-We hope that many users at different scales will try WAL in their environments after upgrading to 1.15 or later and report success or failure so that we can confidently replace BoltDB as the default for new clusters in a future release.
\ No newline at end of file
+We hope that many users at different scales will try WAL in their environments after upgrading to 1.15 or later and report success or failure so that we can confidently replace BoltDB as the default for new clusters in a future release.
+
+## Known Issues
+
+The following issues were discovered after release of Consul 1.15.1. Fixes should be available in the next patch release.
```

Comment:
```suggestion
The following issues were discovered after release of Consul 1.15.1. Fixes should be available in a future patch release.
```

Reply:
Will love to see this come in the next patch release, but I believe this depends on timing!